libcloudforensics/providers/aws/internal/ebs.py

```diff
@@ -305,3 +305,308 @@ def ShareWithAWSAccount(self, aws_account_id: str) -> None:
         },
         OperationType='add'
     )
+
+
+class AWSEBS:
```

Comment:
Might be clearer to call this `AwsEbs`, or `AwsEBS`? Same with other class names.

Reply:
@tomchop Isn't it against the style guide to not keep case for acronyms such as AWS? Back when we introduced AWS (see [here](https://github.com/google/cloud-forensics-utils/pull/28/commits/16c57242c0c4ddf9a9215688aaa1c5ccd7ca0623#r404878599)) we decided to go with uppercase. If we're fine with deviating from the style guide I'm happy to change all impacted classes (FWIW, I also find it not so readable as it is now)

Reply:
Yeah but to acronyms that follow each other are hard to read. There's probably a rule about this somewhere but I can't fin it. Maybe we should go with what makes the most sense?

Reply:
Agreed, happy to push the changes. I'll update similar class names to `AwsXXX`

Reply:
Another option: just EBS. It's already under the AWS provider...